src/app/modules/time-clock/services/entry.service.ts

```diff
@@ -52,8 +45,15 @@ export class EntryService {
   }
 
   stopEntryRunning(idEntry: string): Observable<any> {
-    const url = `${this.baseUrl}/${idEntry}/stop`;
-    return this.http.post(url, null);
+    let path = '';
+
+    if (this.urlInProduction){
+      path = `${this.baseUrl}/${idEntry}/stop`;
+      return this.http.post(path, null);
+    }else{
+      path = `${this.baseUrl}/stop`;
+      return this.http.put(path, null);
+    }
```

Comment:
maybe it could be simplified with ternaries:

```suggestion
    let path = this.urlInProduction ? `${this.baseUrl}/${idEntry}/stop` :  `${this.baseUrl}/stop`;
    return ( this.urlInProduction ? this.http.post(path, null) : this.http.put(path, null) );
```